src/integration-test/java/com/mongodb/jdbc/integration/testharness/IntegrationTestUtils.java

```diff
@@ -553,11 +578,15 @@ private static boolean compareRow(List<Object> expectedRow, ResultSet actualRow)
                     /*
```

Comment:
I'm very unfamiliar with the jdbc rep so I'm most likely missing something, but should this commented section be removed given lines 585-589? If not, [super nit] should we at least modify the commented code so it uses the
```
Object expected_obj = expectedRow.get(i);
Object actual_obj = actualRow.getObject(i + 1);
```
pattern the other sections use?

Reply:
ahah, don't worry, I had a similar question. So, this is/will be addressed by Oliver in one of his PR that's why I left it here.